packages/@aws-cdk/aws-codedeploy/README.md

```diff
@@ -184,3 +184,69 @@ You can also add the Deployment Group to the Pipeline directly:
 // equivalent to the code above:
 deploymentGroup.addToPipeline(deployStage, 'CodeDeploy');
 ```
+
+### Lambda Applications
+
+To create a new CodeDeploy application that deploys to a Lambda function:
+
+```ts
+import codedeploy = require('@aws-cdk/aws-codedeploy');
+
+const application = new codedeploy.LambdaApplication(this, 'CodeDeployApplication', {
+    applicationName: 'MyApplication', // optional property
```

Comment:
Usually, the first code we show should be the complete-minimal-usable example, the "hello world" of this feature. What is the minimal information I need in order to make this work as quickly as possible?

Specifically:
- omit any optional props
- add a minimal deployment group

Then, you can describe all the features in detail.

Reply:
Yeah I think this whole README needs an overhaul.

Reply:
All properties of these classes are optional, so I don't see the point in providing an example that passes nothing - seems the ReadMe would be very verbose if you wanted to tackle each property separately.